Tasks/AzureRmWebAppDeployment/task.loc.json

```diff
@@ -169,10 +168,30 @@
   ],
   "instanceNameFormat": "ms-resource:loc.instanceNameFormat",
   "execution": {
-    "AzurePowerShell": {
-      "target": "$(currentDirectory)\\AzureRMWebAppDeployment.ps1",
-      "argumentFormat": "",
-      "workingDirectory": "$(currentDirectory)"
+    "PowerShell3": {
```

Comment:
You don't have to delete the AzurePowerShell implementation immediately. The old Windows agent will revert the preference order if you set the variable `system.preferazpsoverps3` to `true` on the definition. It may be helpful is someone is totally blocked, then they can revert to the old implementation without having to wait for a hotfix.